Sources/TSPL/TSPL.docc/ReferenceManual/Attributes.md

```diff
@@ -1418,7 +1418,7 @@ that varies between the conditional branches.
 
 For example, the code below defines a simple result builder
 that builds an array of integers.
-This code defines `Compontent` and `Expression` as type aliases,
+This code defines `Component` and `Expression` as type aliases,
```

Comment:
This is the one typo fix that will change the published content — the others are in `@Comment` blocks.